superbuild/projects_modules/music-plugins.cmake

```diff
@@ -10,13 +10,14 @@ function(music_plugins_project)
         medInria
         jsoncons
         asio
-        websocketpp
-        openssl
+		websocketpp
```

Comment:
there are some indentation problems with the changes (it seems a problem with tab and spaces indentation)